retriever/lib/install.py

```diff
@@ -25,7 +27,8 @@ def _install(args, use_cache, debug, compile):
                 if debug:
                     raise
     else:
-        message = "The dataset \"{}\" isn't currently available in the Retriever. Run retriever.datasets() to see a list of currently available datasets".format(args['dataset'])
+        message = "The dataset \"{}\" isn't currently available in the Retriever. Run retriever.datasets() to see a list of currently available datasets".format(
+            args['dataset'])
```

Comment:
Not sure this saves anything since the line is still way over 80 char. If we're going to do something like this we might as well go ahead and split the string as well:

```
        message = "The dataset \"{}\" isn't currently available in the " \
                  "Retriever. Run retriever.datasets() to see a list of" \
                  "currently available datasets".format(args['dataset'])
```